genesis/example/member_test.go

```diff
@@ -107,11 +111,16 @@ func (p *mockProxy) GetReference() object.Reference {
 	return nil
 }
 
+func (p *mockProxy) SetReference(reference object.Reference) {
```

Comment:
Why not inherit mockProxy from mockCallable instead (as with mockFactory, mockParent and mockChild)?